apps/emqx/src/emqx_persistent_message_ds_replayer.erl

```diff
@@ -138,9 +182,23 @@ poll(SessionId, Inflight0, WindowSize) when WindowSize > 0, WindowSize < 16#7fff
         true ->
             %% TODO: Wrap this in `mria:async_dirty/2`?
             Streams = shuffle(get_streams(SessionId)),
-            fetch(SessionId, Inflight0, Streams, FreeSpace, [])
+            fetch(ReplyFun, SessionId, Inflight0, Streams, FreeSpace, [])
     end.
 
+-spec committed_until(track() | marker(), inflight()) -> seqno().
```

Comment:
Nit: maybe add a comment saying that the returned seqno is _exclusive_ (i.e., itself is not committed).  Or _inclusive_, if I got it wrong.  :see_no_evil:

Reply:
34c9c02